cocos/2d/CCScene.cpp

```diff
@@ -186,6 +187,7 @@ void Scene::render(Renderer* renderer)
 #endif
 
     Camera::_visitingCamera = nullptr;
+    FrameBufferObject::applyDefaultFBO();
```

Comment:
Switch frame buffer multiple times.
It is better to cache current frame buffer.